packages/gatsby-remark-images/README.md

```diff
@@ -70,6 +70,7 @@ a reserved keyword `GATSBY_EMPTY_ALT` can be used.
 | `backgroundColor`       | `white` | Set the background color of the image to match the background image of your design.<br /><br />**Note:**<br />- set this option to `transparent` for a transparent image background.<br /> - set this option to `none` to completely remove the image background.                                                                                                                                                                                                                                                                                                                                                                  |
 | `quality`               | `50`    | The quality level of the generated files.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
 | `withWebp`              | `false` | Additionally generate WebP versions alongside your chosen file format. They are added as a srcset with the appropriate mimetype and will be loaded in browsers that support the format. Pass `true` for default support, or an object of options to specifically override those for the WebP files. For example, pass `{ quality: 80 }` to have the WebP images be at quality level 80.                                                                                                                                                                                                                                            |
+| `withAvif`              | `false` | Additionally generate AVIF versions alongside your chosen file format. They are added as a srcset with the appropriate mimetype and will be loaded in browsers that support the format. Pass `true` for default support, or an object of options to specifically override those for the WebP files. For example, pass `{ quality: 80 }` to have the AVIF images be at quality level 80.                                                                                                                                                                                                                                            |
```

Comment:
Pass `true` for default support, or an object of options to specifically override those for the ~WebP files.~ AVIF files.